ui/src/views/Connectors.vue

```diff
@@ -344,13 +344,7 @@
               >
                 <label class="col-sm-3 control-label">{{$t('connectors.filtercheck')}}</label>
                 <div class="col-sm-9">
-                  <input
```

Comment:
If possible, try to not change the logic.

In this case, make sure the code which reads the value just return `enabled` and `disabled` and not the boolean value.